src/SignalR/clients/ts/signalr-protocol-msgpack/src/MessagePackOptions.ts

```diff
@@ -0,0 +1,5 @@
+export interface MessagePackOptions {
```

Comment:
We should put some JSDoc comments on this and it's properties, as it will be new public API (it'll also need API review, @BrennanConroy).

Reply:
Yes, that sounds good. I have updated with JSDoc tags. Let me know if something is missing.